applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_processes.cpp

```diff
@@ -244,7 +244,7 @@ namespace Kratos {
       std::vector<ModelPart::IndexType> elemNodes_1{ 1, 2, 3 };
       model_part.CreateNewElement("IncompressiblePotentialFlowElement2D3N", 1, elemNodes_1, pElemProp);
 
-      auto r_element = model_part.GetElement(1);
+      auto& r_element = model_part.GetElement(1);
```

Comment:
This test was failing because there was a bug in the test! This means that the reference values of the test were wrong.

r_element was not referencing the Element(1) of the model_part so all the definitons that we do of the element below did not appear in the original model part (specifically, r_element.GetValue(WAKE)=1 was only performed at the copy, not at the original model part element)

With the new changes, both point to the same geometry data (as I understand) so r_element.GetValue(WAKE)=1 was correctly setting the element as WAKE in the original model part.

Reply:
@marc can you make a separate PR for the fix please?
This way this PR remains clean